frontend/src/app/(functional-pages)/teams/[id]/page.tsx

```diff
@@ -55,7 +56,12 @@ export default async function Page({ params }: any) {
           Members
         </h3>
         {members.map((m)=> (
```

Comment:
It would probably be more readable to call it member instead of just "m"
```suggestion
        {members.map((member)=> (
```

Reply:
m changed to member and r changed to roles